src/clj_momo/lib/es/document.clj

```diff
@@ -223,6 +224,33 @@
       :result
       (= "deleted")))
 
+(s/defn delete-by-query-uri
+  [uri index-names mapping]
+  (let [index (if (coll? index-names)
```

Comment:
I think it's better to enforce either a collection or a single value, this way the symbol name is not ambiguous